admin/tests/integration/components/organizations/places_test.js

```diff
@@ -68,6 +68,30 @@ module('Integration | Component | Organizations | Places', function (hooks) {
         assert.dom(screen.queryByText('Aucun résultat')).doesNotExist();
         assert.dom(screen.getByText('7777')).exists();
       });
+
+      module('display delete places lot modal', function () {
+        test('it should not display button to delete places lot', async function (assert) {
```

Comment:
**question** : je comprends pas ce test, `should not display` mais  on `click` et ça `exists`...

Reply:
mauvais nom de it.